wd_helper.au3

```diff
@@ -2107,15 +2107,16 @@ EndFunc   ;==>__WD_ElementBuildActionString
 Func __WD_ErrHnd()
 
 EndFunc   ;==>__WD_ErrHnd
+
 ; #INTERNAL_USE_ONLY# ===========================================================================================================
 ; Name ..........: __WD_JsonButtonAction
 ; Description ...: Create JSON string for button actions
 ; Syntax ........: __WD_JsonButtonAction($iButton, $sAction)
-; Parameters ....: $iButton             - an integer value.
-;                  $sAction             - a string value.
+; Parameters ....: $iButton             - an integer value declared in _WD_ElementActionEx
+;                  $sAction             - a string value declared in _WD_ElementActionEx
```

Comment:
I don't believe this should be added. What if we decide to call this function from a different function?

Reply:
Then we will think how to change them.

Reply:
I think you are saying to leave it as-is. If so, then I disagree. 

IMO, it would be better to provide a proper description of the parameter instead of referencing the calling function.

Reply:
So maybe something like this:
```AutoIt
; Parameters ....: $iButton             - an integer value 0 is .... 1 ... 2 ....
;                  $sAction             - a string value - action type that should be performed
```

Reply:
btw. what about 
`; Link ..........:`
Do you have any link to reference documentation ?
I'm courious how you find out what `$iButton` means and what kind of $sAction can be taken.

Reply:
This [link](https://www.w3.org/TR/webdriver/#actions) is already present in header of _WD_Action, so I don't see the need to add it elsewhere.

Reply:
Fixed:
https://github.com/mlipok/WebDriver/commit/3d3c9454ad9f6f446c2c16deb7e4a23d57bbbaab